packages/experimental/Shadow/SPEC.md

```diff
@@ -48,9 +48,10 @@ If `undefined` is passed into the `shadowToken` prop, no shadow will be rendered
 
 ## Implementation details
 
-In order to get around the React Native restriction of one Shadow per View, the Shadow component adds an extra View around the original View. One of the shadows is placed on the original View and the other is placed on the extra View.  The Shadow component smartly extracts the original View's style props so that the extra View isn't noticable (ex. will always be the same size as the original View, padding/margins are set correctly, etc.)
+In order to get around the React Native restriction of one Shadow per View, the Shadow component adds an extra View around the original View. One of the shadows is placed on the original View and the other is placed on the extra View. The Shadow component smartly extracts the original View's style props so that the extra View isn't noticable (ex. will always be the same size as the original View, padding/margins are set correctly, etc.)
 
 ## Notes
 
+- Known issue: The Shadow component does not currently work if you are wrapping an Animated component.
```

Comment:
Is this something we'd want to eventually support? If so can you file a bug & assign to me

Reply:
I don't know if it is. It might be possible to work around it, but I intentionally kept that out of this PR. If/when we start Switch for iOS, I think we can revisit.